.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,64 @@
+          upload_sources: true
+          upload_translations: false
+          download_translations: false
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: false
+          
+        env:
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+          
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)